src/keyboard.jl

```diff
@@ -23,15 +23,15 @@ using .MouseButtons
 # Thus user code to check if X is pressed is written as
 #       ` if game.keyboard.X`
 const singleCharStrings = string.(collect('a':'z'))
-struct KeyHolder; end
-const Keys = KeyHolder()
-function Base.getproperty(k::KeyHolder, s::Symbol)
+struct KeyHolder{T} end
+const Keys = KeyHolder{:SDLK_}()
+const KeyMods = KeyHolder{:KMOD_}()
```

Comment:
Sorry, I don't understand what is happening here? Can you clarify please?

Reply:
use the trick of "Value type" 
https://docs.julialang.org/en/v1/manual/types/#%22Value-types%22

In addition to type, Symbol or bits_value were also allowed in julia parametric Type. Therefore compiler can compile/dispatch based on the symbol or value